server.py

```diff
@@ -17,67 +15,10 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 
-def check_auth(username, password):
-    """This function is called to check if a username /
-    password combination is valid.
-    """
-    return username == settings.RECEIPT_USER and password == settings.RECEIPT_PASS
-
-
-def authenticate():
-    """Sends a 401 response that enables basic auth"""
-    # TODO: return appropriate auth error xml
-    resp = make_response(render_template('error.xml'), 401)
-    resp.headers['Content-Type'] = 'application/vnd.ons.error+xml'
-    return resp
-
-
-def requires_auth(f):
-    @wraps(f)
-    def decorated(*args, **kwargs):
-        auth = request.authorization
-        if not auth or not check_auth(auth.username, auth.password):
-            return authenticate()
-        return f(*args, **kwargs)
-    return decorated
-
-
-@app.errorhandler(400)
-def not_found(error):
-    resp = make_response(render_template('error.xml'), 400)
-    resp.headers['Content-Type'] = 'application/vnd.ons.error+xml'
-    return resp
-
-
 @app.route('/receipts', methods=['POST'])
 def receipt():
-    return make_response('{"status": "ok"}', 200)
-
-
-@app.route('/reportingunits/<ru_ref>/collectionexercises/<exercise_sid>/receipts', methods=['POST'])
```

Comment:
I have no idea where any of this ever got used, and I can't think of a way to check it, but it looks okay? 
Do you want to update the readme as this is removed?